tests/lib/cli.js

```diff
@@ -570,21 +570,21 @@ describe("cli", () => {
 
                 it(`should only print error with configType:${configType}`, async () => {
                     const filePath = getFixturePath("single-quoted.js");
-                    const cliArgs = `--no-ignore --quiet  -f compact --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
+                    const cliArgs = `--no-ignore --quiet  -f stylish --rule 'quotes: [2, double]' --rule 'no-unused-vars: 1' ${filePath}`;
 
                     await cli.execute(cliArgs, null, useFlatConfig);
 
                     sinon.assert.calledOnce(log.info);
 
                     const formattedOutput = log.info.firstCall.args[0];
 
-                    assert.include(formattedOutput, "Error");
-                    assert.notInclude(formattedOutput, "Warning");
+                    assert.include(formattedOutput, "1 error");
+                    assert.include(formattedOutput, "0 warnings");
                 });
 
                 it(`should print nothing if there are no errors with configType:${configType}`, async () => {
                     const filePath = getFixturePath("single-quoted.js");
-                    const cliArgs = `--quiet  -f compact --rule 'quotes: [1, double]' --rule 'no-unused-vars: 1' ${filePath}`;
+                    const cliArgs = `--quiet  -f stylish --rule 'quotes: [1, double]' --rule 'no-unused-vars: 1' ${filePath}`;
```

Comment:
```suggestion
                    const cliArgs = `--no-ignore --quiet -f stylish --rule 'quotes: [1, double]' --rule 'no-undef: 1' ${filePath}`;
```

Similar to the previous.